1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,63 @@
+import { fizzbuzz } from "./fizzbuzz"
 
 describe("fizzbuzz", () => {
+    it("returns a string", () => {
+        const result = fizzbuzz(4);
+        expect(typeof result).toBe("string");
+    });
 
+    it("returns 1 as a string", () => {
+        const result = fizzbuzz(1);
+        expect(result).toEqual("1");
+    });
```

Comment:
You can consolidate the functions into one, using `test.each`.
https://jestjs.io/docs/api#testeachtablename-fn-timeout

```js
test.each([3, 27])( "should return 'Fizz' when the number is devisable by 3, such as %i", number => { 
  expect(fizzBuzz(number)).toBe('Fizz');
});`
```

Same can be said to the other tests 👍